userspace/libscap/scap_bpf.c

```diff
@@ -786,7 +786,7 @@ static int32_t populate_syscall_table_map(scap_t *handle)
 		}
 	}
 
-	return bpf_map_freeze(handle->m_bpf_map_fds[SYSDIG_SYSCALL_TABLE]);
+	return SCAP_SUCCESS;
```

Comment:
Do not freeze syscall_table_map as we use it in `scap_bpf_set_simple_mode` and `scap_bpf_handle_event_mask` that will be called at runtime (ie: with an initialized bpf probe).